### ISSUE TITLE
the property IPaymentResponseAction.Type Does not get deserialized

### ISSUE THREAD

**Issue by akScript** [Adyen/adyen-dotnet-api-library#374]
**Bug description:**
Using Web Drop-in integration, After [Making a payment](https://docs.adyen.com/online-payments/drop-in-web#step-3-make-a-payment) that needs additional front-end actions, the PaymentResponse.Action Property needs to be returned to the front-end, however, the property IPaymentResponseAction.Type does not get deserialized, causing the frontend to throw an Invalid Action Error. I think that is because the Type property does not have a DataMember attribute.

**Steps to reproduce:**
1- implement steps on Web Drop-in integration guide
2- [Make a payment](https://docs.adyen.com/online-payments/drop-in-web#step-3-make-a-payment) using a 3DS 2 test card for instance (Visa | 4917 6100 0000 0000 | 03/2030 | 737)
3- return paymentResponse.Action property to the frontend
4- see error: Invalid Action

Adyen .net Package version: 6.0.0
@adyen/adyen-web version : "3.20.0"

**Comment by AlexandrosMor** [comment]
Hello @akScript,
Thank you for raising this issue. Do you mean the type field in [action](https://github.com/Adyen/adyen-dotnet-api-library/blob/develop/Adyen/Model/Checkout/Action/CheckoutAwaitAction.cs#L45) classes ? 
We added the type data member annotation in develop. Could you maybe check if it works for you and let us know? 

kind regards,
Alexandros

**Comment by akScript** [comment]
Hello @AlexandrosMor,
thank you for the quick response, yes I mean the classes that implement the `IPaymentResponseAction ` interface, I Just check the developed branch, and indeed the type property is being returned now, however, I still get an Invalid Action
```
Error: Invalid Action
    at t.handleAction (adyen.js:1)`
```
there is the whole action object:

```
  "action": {
    "type": "threeDS2Fingerprint",
    "paymentData": "Ab02b4c0!BQABAgBNSfsOs...",
    "paymentMethodType": "scheme",
    "token": "eyJ0aHJlZURTTWVzc2FnZ..."
  }
```

**Comment by AlexandrosMor** [comment]
Hello @akScript,

Thank you for testing. Just to clarify it so the develop is fixing this issue but not the #373, is it correct ? 

regards,
Alexandros

**Comment by akScript** [comment]
Hello @AlexandrosMor
Thank you for your response, yes I mean the classes that implement the `IPaymentResponseAction` interface, and indeed I'm now getting back the type property with the action object.

Correct this issue seems to be fixed, but not #373

**Comment by AlexandrosMor** [comment]
Hello @akScript,
 
The fix is included in the new release [6.1.0 ](https://github.com/Adyen/adyen-dotnet-api-library/releases/tag/6.1.0)
If you face any new issues please let us know to help you further

kind regards,
Alexandros
